.bazelci/build_bazel_binaries.yml

```diff
@@ -49,3 +49,13 @@ platforms:
     build_targets:
       - "//src:bazel.exe"
       - "//src:bazel_nojdk.exe"
+  windows_arm64:
```

Comment:
We don't support `windows_arm64` on our CI yet because we don't have the arm64 machine, but we can cross-compile a Windows arm64 binary just like we did for macos arm64. Have you tried to cross-compile a arm64 Bazel binary, does it work?

Reply:
Yes, cross-compilation should work.